openprescribing/frontend/models.py

```diff
@@ -85,6 +105,8 @@ class PCT(models.Model):
     )
     code = models.CharField(max_length=3, primary_key=True,
                             help_text='Primary care trust code')
+    regional_team = models.ForeignKey(RegionalTeam, null=True)
```

Comment:
I presume you've already thought through why we allow nulls here - might be worth adding a comment explaining.

I want to be sure we don't forget to add STP membership when new CCGs are created.